standard/expressions.md

```diff
@@ -6195,6 +6212,55 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 >
 > *end example*
 
+### §ref-assignment-new-clause Ref assignment
+
+The left operand shall be an expression that binds to a ref local variable, a ref parameter (other than `this`), or an out parameter. The right operand shall be an expression that yields a *variable_reference* designating a value of the same type as the left operand.
```

Comment:
I think it would be good to have an introductory sentence to give context. Just something as simple as "The `= ref` operator is known as the *ref assignment* operator." Just something to ease the reader back into the context.

Reply:
I made that change. I think I like it. 

Worth discussion in our meeting:  Should `= ref` be described as a modifier on the right operand of an assignment, or a new operator?

Reply:
Action 2023-04-19: It's a different operator. @BillWagner  to check that it's listed a) in the table of all operators; b) in the grammar.

Reply:
verified and updated.